app/assets/javascripts/miq_tree.js

```diff
@@ -471,6 +471,7 @@ window.miqInitTree = function(options, tree) {
   }
 
   // Pre-process partially checkbox state for parent nodes
+  // TODO make function for it - covnersion time
```

Comment:
what does converting time have to do with converting checkboxes? (stray TODO comment?)

Reply:
It's still work in progress, I have to complete all the functionality before be able to merge

Reply:
But yes, it shouldn't be in that commit :/

Reply:
Oh, sorry :) I saw the review request and didn't notice the wip, don't worry about it :)